CONTRIBUTING.md

```diff
@@ -37,27 +97,38 @@ To work on the codebase for this project, you will need to have those dependenci
 <details>
 	<summary>Click to Expand</summary>
 
-* After forking this repository to your own github account, and cloning it to your dev environment, you can now create a new branch on your machine. It's wise to name this branch, after the issue you are trying to fix, or the feature you are trying to add.
+- After forking this repository to your own github account, and cloning it to
+  your dev environment, you can now create a new branch on your machine. It's
+  wise to name this branch, after the issue you are trying to fix, or the
+  feature you are trying to add.
 
-	```bash
-	git checkout -b creatingContributionGuide
-	```
+      	```bash
+      	git checkout -b creatingContributionGuide
+      	```
 
-* In the example above, I have created a new branch, named "creatingContributionGuide". This command also "checks out" the branch, meaning git now knows that is the branch you are working on. You can check what branch you are working on by using the `branch` command.
+- In the example above, I have created a new branch, named
+  "creatingContributionGuide". This command also "checks out" the branch,
+  meaning git now knows that is the branch you are working on. You can check
+  what branch you are working on by using the `branch` command.
 
-	```bash
-	git branch
-	```
-* Following my example, `git branch`, would output "creatingContributionGuide" in my terminal.
+      	```bash
+      	git branch
+      	```
 
-* Once you have finished working on your issue, push your changes to your own github repo, and then submit a pull request.
+- Following my example, `git branch`, would output "creatingContributionGuide"
+  in my terminal.
 
-* To return to your main `master` branch, type the following command in your terminal.
+- Once you have finished working on your issue, push your changes to your own
+  github repo, and then submit a pull request.
 
-	```bash
-	git checkout master
-	```
-</details>
+- To return to your main `master` branch, type the following command in your
+  terminal.
+
+      ```bash
+      git checkout master
+      ```
+
+  </details>
 
 Before committing, please lint your code:
```

Comment:
Below this line there's an example that asks the reader to lint their code by installing flake8. This is unnecessary. The command to lint your code is `make lint`

Reply:
Done.